README.md

```diff
@@ -78,12 +78,12 @@ It is recommended to install at least 16GB RAM in your build system.
 
 ### Build System
 * CMake 3.15 or newer is required. [Download](https://cmake.org/download/) and install proper one if the cmake is older than 3.15.
-* Ninja is requred.
+* Ninja is required.
 
 ### Install Dev and Tools Packages
 #### Ubuntu
 ```
-sudo apt-get install build-essential curl g++-multilib gcc-multilib git pkg-config python3
+sudo apt-get install build-essential curl g++-multilib gcc-multilib git ninja-build pkg-config python3
```

Comment:
The build tools had been required above.
But this change is fine to me, I think we could add 'cmake' too.

Reply:
Good point! I added it (and removed the RHEL7 line).